pkg/apis/batch/types.go

```diff
@@ -267,11 +267,13 @@ type CronJobSpec struct {
 
 	// The number of successful finished jobs to retain.
 	// This is a pointer to distinguish between explicit zero and not specified.
+	// Defaults to 3.
```

Comment:
I'm not sure if I need to issue any generation command since I modified this to explicit the default.